src/components/Dialog-Window/ExternalResources/ExternalResources.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from "react";
+import styles from "./ExternalResources.module.scss";
+
+export type DialogExternalResourcesProps = {
+  url: string;
+};
+
+export const DialogExternalResources: React.FC<
+  DialogExternalResourcesProps
+> = ({ url }) => {
+  return (
+    <div className={styles.wrapper}>
+      <iframe src={url} frameBorder="0" title="External Resrouces" />
```

Comment:
I suggest adding a label field 

```suggestion
  url: string;
  label: string;
};

export const DialogExternalResources: React.FC<
  DialogExternalResourcesProps
> = ({ url, label }) => {
  return (
    <div className={styles.wrapper}>
      <iframe src={url} frameBorder="0" title={label} />
```

Or translating the title

```suggestion
  url: string;
};

export const DialogExternalResources: React.FC<
  DialogExternalResourcesProps
> = ({ url }) => {
  // TODO: Translate
  const iframeTitle = "External resource";

  return (
    <div className={styles.wrapper}>
      <iframe src={url} frameBorder="0" title={iframeTitle} />
```